lib/player.rb

```diff
@@ -50,5 +51,24 @@ def highest_word_score
       highest_scoring_word = ScrabbleMod::Scrabble.highest_score_from(@plays)
       return ScrabbleMod::Scrabble.score(highest_scoring_word)
     end
+
+    def tiles
```

Comment:
It seems like you could use an `attr_reader` for the `@player_tiles` instance variable rather than creating an additional method for this